File: Php/app/Urls/CashgameMatrixUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameYearUrlModel;
	use entities\Homegame;

	class CashgameMatrixUrlModel extends HomegameYearUrlModel{

		public function __construct(Homegame $homegame, $year){
			parent::__construct(RouteFormats::cashgameMatrix, RouteFormats::cashgameMatrixWithYear, $homegame, $year);
		}

	}

}

File: Web/Models/Url/CashgameMatrixUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.Url{

	class CashgameMatrixUrlModel : HomegameYearUrlModel{

	    public CashgameMatrixUrlModel(Homegame homegame, int year) : base(RouteFormats.CashgameMatrix, RouteFormats.CashgameMatrixWithYear, homegame, year)
	    {
	        
	    }

	}

}